### DISCUSSION TITLE
Differences between a model and modelPlus

### DISCUSSION THREAD

**Discussion by zmce2018** [timeseriesAI/tsai#537] [Q&A] [answered]
Hi Everyone, 

I have been running experience with different models. Previously, I have been testing models that predict the single-step output. I am now working on the models that predict the multi-step output. My question is do I have to use modelPlus version for the multi-step output?  

If that is the case, is it easy for me to convert the model into modelPus version? I have found gMLP very powerful but it doesn't have gMLPPluse available yet. 

Many thanks in advance.

**Comment by marcuswang6** [comment]
I have the same question!

**Comment by marcuswang6** [comment]
I have the same question!

**Comment by oguiza** [comment, marked as answer]
Hi @zmce2018 and @Waterkin , cc:@radi-cho
If you are interested you can compare any model with a Plus version with its stripped-down version and learn the differences. It's not too difficult. You may consider creating a PR to add a gMLPPlus version. That'd be great, and I can help if you find any issues.
